deformable_manipulation_experiment_params/launch/generic_experiment_mengyao.launch

```diff
@@ -11,11 +11,12 @@
     <arg name="feedback_covariance"                     default="0.0"/>
     <arg name="start_bullet_viewer"                     default="true"/>
     <arg name="screenshots_enabled"                     default="false"/>
+    <arg name="rope_gripper_apperture"                  value="0.03"/>
```

Comment:
With a value set, this is not an input to the launch file, lets move it to somewhere else that makes sense. In addition, does it need to be an arg, or just a param? I.e. are you using it as a input to the launch file or doing math with it?